python/ray/serve/api.py

```diff
@@ -920,14 +899,6 @@ def create_backend(
             mapping strings to values for the following supported options:
             - "num_replicas": number of processes to start up that
             will handle requests to this backend.
-            - "max_batch_size": the maximum number of requests that will
-            be processed in one batch by this backend.
-            - "batch_wait_timeout": time in seconds that backend replicas
-            will wait for a full batch of requests before processing a
-            partial batch.
-            - "max_concurrent_queries": the maximum number of queries that
```

Comment:
"max_concurrent_queries" wasn't removed in the other docstrings, should it be removed here?

Reply:
oops! Good catch!